frontend/cypress/e2e/vessel_sidebar_reporting.spec.ts

```diff
@@ -123,4 +123,39 @@ context('Vessel sidebar reporting tab', () => {
     cy.get('*[data-cy="side-window-current-reportings"]').should('have.length', 3)
     cy.get('*[data-cy="side-window-current-reportings"]').last().contains('FRAIS AVIS MODE')
   })
+
+  it('An observation reporting should be modified to an Infraction suspicion', () => {
+    // Given
+    cy.get('*[data-cy="vessel-search-input"]', { timeout: 10000 }).type('RENCONTRER VEILLER')
```

Comment:
Just fyi I created a `cy.fill()` command to accessibilty-fill inputs and avoid adding `data-cy` everywhere ^^. It can be improved by the way to fill any type of input.